crates/next-core/src/next_app/app_page_entry.rs

```diff
@@ -174,7 +174,9 @@ async fn wrap_edge_page(
         },
         fxindexmap! {
             "sriEnabled" => serde_json::Value::Bool(sri_enabled).to_string().into(),
-            "nextConfig" => serde_json::to_string(next_config)?.into(),
+            // TODO do we really need to pass the entire next config here?
+            // This is bad for invalidation as any config change will invalidate this
```

Comment:
Yeah, I think the assumption was that it doesn't change frequently, and the process lifetime of turbopack was short, so it didn't really matter. With a persistent cache, this sort of thing now feels more important.

It's going to suck to refactor some of this stuff to only pass down the subset of options it needs though.